src/applications/ivc-champva/10-10D/tests/unit/helpers/helpers.unit.spec.js

```diff
@@ -91,3 +90,39 @@ describe('getTopLevelFormData helper', () => {
     ).to.not.be.undefined;
   });
 });
+
+describe('populateFirstApplicant', () => {
+  const newAppInfo = {
+    name: { first: 'First', last: 'Last' },
+    email: 'fake@va.gov',
+    phone: '1231231234',
+    address: { street: '123 st' },
+  };
+  it('Should add an applicant to the start of `formData.applicants` array', () => {
+    const formData = { applicants: [{ applicantName: { first: 'Test' } }] };
+    const result = populateFirstApplicant(
+      formData,
+      '',
+      newAppInfo.name,
+      newAppInfo.email,
+      newAppInfo.phone,
+      newAppInfo.address,
+    );
+    expect(result.applicants.length).to.equal(2);
+    expect(result.applicants[0].applicantName.first).to.equal(
+      newAppInfo.name.first,
+    );
+  });
+  it('Should add the applicants array if it is undefined', () => {
+    const formData = {};
+    const result = populateFirstApplicant(
+      formData,
+      '',
+      newAppInfo.name,
+      newAppInfo.email,
+      newAppInfo.phone,
+      newAppInfo.address,
+    );
+    expect(result.applicants.length).to.equal(1);
+  });
+});
```

Comment:
This tests our function used to prefill the first applicant slot with the certifier's info (if user selected `applicant`)